op-service/eth/blob_test.go

```diff
@@ -72,7 +116,18 @@ func TestTooLongDataEncoding(t *testing.T) {
 	data := Data(make([]byte, BlobSize))
 	var b Blob
 	err := b.FromData(data)
-	if err == nil {
-		t.Errorf("expected error, got none")
-	}
+	require.Error(t, err)
+}
+
+func FuzzEncodeDecodeBlob(f *testing.F) {
```

Comment:
You will need to add a call to run this fuzz test in the makefile under `fuzz`

https://github.com/ethereum-optimism/optimism/blob/develop/op-service/Makefile

Reply:
done